sql/src/main/java/io/crate/analyze/AnalyzedColumnDefinition.java

```diff
@@ -70,16 +70,19 @@
     private Expression generatedExpression;
     private final static Set<String> NO_DOC_VALUES_SUPPORT = Sets.newHashSet("object", "geo_shape");
 
-    public AnalyzedColumnDefinition(@Nullable AnalyzedColumnDefinition parent) {
-        this.parent = parent;
-    }
-
-    public void name(String name) {
+    public static void checkName(String name) {
```

Comment:
Would rename to validateName.
Could be private and moved after the public methods in the class.